example/spec/testAPI/APISuccessSpec.js

```diff
@@ -158,6 +158,21 @@ describe('The Cumulus API', () => {
     });
   });
 
+  describe('executions endpoint', () => {
+    it('returns tasks metadata with name and version', async () => {
+      const executionResponse = await apiTestUtils.getExecution({
+        prefix: config.stackName,
+        arn: workflowExecution.executionArn
+      });
+      expect(executionResponse.tasks.length).not.toEqual(0);
```

Comment:
So this test is technically passing for me. But only because undefined != 0
But I'm finding the executionResponse.tasks to be empty.

Reply:
I'll update it to expect isDefined as well.